app/javascript/components/pages/FateIndex.js

```diff
@@ -1,13 +1,39 @@
-import React,{Component} from 'react'
+import React, { Component } from "react";
+import { Col, Card, CardSubtitle } from "reactstrap";
 
-class FateIndex extends Component{
-    render() {
-
-        return(
+class FateIndex extends Component {
+  //this page still needs work
+  render() {
+            console.log(this.props.events)
+    return (
+      <>
+      <div>
+          <h2>This is Fate Index Page Test1</h2>         
+            {this.props.events && this.props.events.map(event => {
```

Comment:
This can be simplified to use option chaining to achieve the same effect `this.prop.events?.map(` (just a not. not a "you should change this" both work just fine.

Reply:
is there any reasoning to use one over the other so moving forward i know which to use? or is it just considered common practice to use the ? to decide?